Cross.toml

```diff
@@ -3,3 +3,5 @@ passthrough = [
 	"RUSTFLAGS",
 	"RUST_TEST_THREADS"
 ]
+[target.loongarch64-unknown-linux-gnu]
+image = "ghcr.io/loongarchlinux/archlinux:latest"
```

Comment:
Add the following line to specify the custom target, or it will fall back to Amd64/Linux:

```
image.toolchain = ["loongarch64-unknown-linux-gnu"]
```